subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/verification/DependencyVerificationSignatureCheckIntegTest.groovy

```diff
@@ -1420,6 +1431,39 @@ One artifact failed verification: foo-1.0.jar (org:foo:1.0) from repository mave
         noExceptionThrown()
     }
 
+    def "can disable reaching out to key servers"() {
+        createMetadataFile {
+            keyServer(keyServerFixture.uri) // make sure we declare a key server for tests
+            disableKeyServers() // but disable access to the key server
+            verifySignatures()
+            addTrustedKey("org:foo:1.0", validPublicKeyHexString)
+            addTrustedKey("org:foo:1.0", validPublicKeyHexString, "pom", "pom")
+        }
+
+        given:
+        javaLibrary()
+        uncheckedModule("org", "foo", "1.0") {
+            withSignature {
+                signAsciiArmored(it)
+            }
+        }
+        buildFile << """
+            dependencies {
+                implementation "org:foo:1.0"
+            }
+        """
+
+        when:
+        fails ":compileJava"
+
+        then:
+        failure.assertHasCause """Dependency verification failed for configuration ':compileClasspath'
```

Comment:
The fact that reaching out to the key server has been disabled should be printed out in this error message.